### DISCUSSION TITLE
Help for Ukraine

### DISCUSSION THREAD

**Discussion by krisztiannn2004** [meshtastic/Discussions#101] [General]
Hi Im from Ukraine and I ask for meshtastic devices for emergency communications if you have heltec v3s (or anything) you dont need laying around and you can send it to Ukraine or Hungary pls contact me

**Comment by mariotti** [comment]
Hi @krisztiannn2004 

It would be nice if you can provide a more official address and eventually more details.

Me myself, I already do donation for Ukraine, I assume also some of other users.

But why should we contact you?
